src/whylogs/app/logger.py

```diff
@@ -355,6 +364,82 @@ def log_image(self,
 
         track_image(self._profiles[-1]["full_profile"])
 
+    def log_local_dataset(self, root_dir, folder_feature_name="folder_feature", image_feature_transforms=None, show_progress=False):
+        """
+        Log a local folder dataset
+        It will log data from the files, along with structure file data like
+        metadata, and magic numbers. If the folder has single layer for children
+        folders, this will pick up folder names as a segmented feature
+
+        Args:
+            root_dir (str): directory where dataset is located.
+            folder_feature_name (str, optional): Name for the subfolder features, i.e. class, store etc.
+            v (None, optional): image transform that you would like to use with the image log
+
+        Raises:
+            NotImplementedError: Description
+        """
+        try:
+            from PIL.Image import Image as ImageType
+        except ImportError as e:
+            ImageType = None
+            logger.debug(str(e))
+            logger.debug(
+                "Unable to load PIL; install Pillow for image support")
+
+        if show_progress:
+            try:
+                from tqdm import tqdm
+            except ImportError as e:
+                show_progress = False
+                logger.debug(str(e))
+                logger.debug(
```

Comment:
```suggestion
                logger.warning(
```
Warning is more appropriate here since the flag is explicitly set